app/views/admin/loan_questions/_loan_types.slim

```diff
@@ -0,0 +1,31 @@
+= f.input "loan_types" do
+  - if @loan_question.parent
+    / TODO: Use simple form yml to add contextual text for options
+    = f.input_field :override_associations, as: :select
+
+  / TODO: Table should be hidden or shown when modal first opens based on override_associations
+  table.table.loan-types-table
+    thead
+      tr
+        th colspan="2" Loan type
+        th Threshold Amount
+    tbody
+      = f.simple_fields_for :custom_field_requirements, @requirements do |sf|
+        / CustomFieldRequirement: sf.object
+        / Not all custom field requirements have option_ids
+        / It appears that those without are not loan types
+        / TODO: Move this logic to controller and remove any unneeded comments
+        - if sf.object.option_id
+          tr
+            td
+              input.require-checkbox type="checkbox" checked=!sf.object.new_record?
```

Comment:
This input value is not hooked up in a meaningful way to the form. Keys should reflect what is needed to save to the form.